src/components/Panel/Rightside/Members.vue

```diff
@@ -91,31 +92,64 @@ export default {
 
   data () {
     return {
-      userQuery: null,
+      query: null,
+
+      members: [],
+      nonMembers: [],
     }
   },
 
   computed: {
     ...mapGetters({
       users: 'users/list',
     }),
+  },
 
-    members () {
-      return this.users.filter(u => this.channel.isMember(u.userID))
+  watch: {
+    'query' (query) {
+      if (query) {
+        const nonMembers = []
+        const limit = 15
+        this.$SystemAPI.userList(({ query, limit })).then(({ set }) => {
+          (set || nonMembers).forEach(u => {
+            if (!this.channel.isMember(u.userID) && u.userID !== this.$auth.user.userID) {
+              nonMembers.push(new User(u))
+            }
+          })
```

Comment:
Even tho, this is safe by the `.forEach` documentation, this feels wrong to me and there is probably a browser version, that will not work as intended.

Just use `(set || [])`, or eaven better:
```js
        this.$SystemAPI.userList(({ query, limit })).then(({ set = [] }) => {
          set.forEach(u => {
```